src/Chapter2.hs

```diff
@@ -620,7 +636,9 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven [] = []
+takeEven (x : xs) = x : takeEven (drop 1 xs)
```

Comment:
Nice usage of standard function `drop`! Though, you can pattern-match on the first two elements of the list first and avoid using `drop` at all